napari/utils/notebook_display.py

```diff
@@ -127,4 +132,109 @@ def _repr_html_(self):
         return f'<img src="{url}" alt="{_alt}"></img>'
 
 
+class ExportView:
```

Comment:
I'm not 100% that we need the notebook display for this...? Maybe it's fine, but it just adds a lot of uncovered lines of code...

Reply:
(It could come in a subsequent PR, maybe)

Reply:
removed for now